api/nodes/views.py

```diff
@@ -2889,6 +2889,9 @@ def get_default_queryset(self):
     def get_queryset(self):
         return self.get_queryset_from_request()
 
+    def perform_create(self, serializer):
+        return serializer.save(node=self.get_node())
```

Comment:
Is `node` used in `NodeWikiSerializer#save`?

Reply:
See http://www.django-rest-framework.org/topics/3.0-announcement/#changes-to-prepost-save-hooks and http://www.django-rest-framework.org/api-guide/serializers/#saving-instances. 

`perform_create` calls `serializer.save()` which passes `node` to `NodeWikiSerializer.create()`